core/app/models/spree/shipment.rb

```diff
@@ -222,7 +222,8 @@ def inventory_units_for(variant)
     end
 
     def to_package
-      package = Stock::Package.new(stock_location, order)
+      package_factory = Spree::Stock::Coordinator.new(self).packer(stock_location).package_factory
```

Comment:
Is there any reason why don't get the factory from `Spree::Config.package_factory`?

Reply:
no, I changed it.